src/miner.cpp

```diff
@@ -473,6 +473,9 @@ void static BitcoinMiner(CWallet *pwallet)
     );
 
     try {
+        unique_ptr<equi> peq;
+        if (solver == "tromp")
+            peq.reset(new equi(1));
```

Comment:
anyway why do you change the above line? it's working for me in that way (and in this way has no memory leak:-)